src/sentry/integrations/slack/views/link_team.py

```diff
@@ -173,6 +209,8 @@ def handle(self, request: Request, signed_params: str) -> HttpResponseBase:
 
         if not created:
             message = ALREADY_LINKED_MESSAGE.format(slug=team.slug)
+            _logger.info("team.already_linked", extra=logger_params)
```

Comment:
Is this a useful log?

Reply:
good call, its unactionable, Ill remove it